packages/neuron-wallet/src/block-sync-renderer/task.ts

```diff
@@ -1,37 +1,45 @@
-import { ipcRenderer } from 'electron'
 import initConnection from 'database/chain/ormconfig'
 import { Address as AddressInterface } from 'database/address/address-dao'
 import Queue from './sync/queue'
 import { register as registerTxStatusListener, unregister as unregisterTxStatusListener } from './tx-status-listener'
 import logger from 'utils/logger'
 import { LumosCellQuery } from './sync/indexer-connector'
+import { expose } from 'utils/worker'
+import env from 'env'
 
 let syncQueue: Queue | null
 
-ipcRenderer.on('block-sync:start', async (_, url: string, genesisHash: string, addressesMetas: AddressInterface[]) => {
-  if (syncQueue) {
-    await syncQueue.stopAndWait()
+export type SyncTask = typeof syncTask
+
+const syncTask = {
+  unmount() {
+    unregisterTxStatusListener()
+
+    logger.info("Sync:\tstop block sync queue")
+    syncQueue?.stop()
+    syncQueue = null
+  },
+  async start(url: string, genesisHash: string, addressesMetas: AddressInterface[]) {
+    if (syncQueue) {
+      await syncQueue.stopAndWait()
+    }
+
+    // reset `fileBasePath` from master process
+    env.fileBasePath = process.env['fileBasePath'] ?? env.fileBasePath
```

Comment:
Where does this `fileBasePath` value come from originally?

Reply:
From master process:
https://github.com/nervosnetwork/neuron/blob/eddb4e4b78053ba81288b9bd9416f99b52a4434f/packages/neuron-wallet/src/block-sync-renderer/index.ts#L87-L93

Reply:
Will this be changed when it switches to different networks?

Reply:
It won't changed.
`env.fileBasePath` should stay intact when Electron starts, but since there is no `electron.app` or `remote.app` module in the child process, the child process's `env.fileBasePath` is actually a fake value for passing the test.
https://github.com/nervosnetwork/neuron/blob/5a7f78d5ec4d674b035140fe6944e8d4eeab8120/packages/neuron-wallet/src/env.ts#L6-L16